hex.html

```diff
@@ -11,21 +11,24 @@
   <body>
     <nav>
       <div class="nav-center">
-        <h4>Color Flipper</h4>
+        <h4 id="header-text">Random Color Generator</h4>
         <ul class="nav-links">
           <li>
             <a href="index.html">Simple</a>
           </li>
           <li>
             <a href="hex.html">Hex</a>
           </li>
+          <li>
+            <a href="./rgb.html">Rgb</a>
+          </li>
         </ul>
       </div>
     </nav>
 
     <main>
       <div class="container">
-        <h2>Background color : <span class="color">#f1f5f8</span></h2>
+        <h2>Color : <span class="color">#f1f5f8</span></h2>
```

Comment:
Maybe you can put something like `"Click the button to start"` instead a fixed color?